### DISCUSSION TITLE
dav1d error on homebrew apple silicon when running ffmpeg

### DISCUSSION THREAD

**Discussion by dbz2k** [Homebrew/discussions#2705] [Everyday usage] [answered]
When running ffmpeg I get this error.
`dyld: Library not loaded: /opt/homebrew/opt/rav1e/lib/librav1e.0.5.1.dylib
  Referenced from: /opt/homebrew/bin/ffmpeg
  Reason: image not found
Abort trap: 6`

When trying to reinstall dav1d I get this.
`Error: Could not get runtime dependencies from /opt/homebrew/opt/ffmpeg/.brew/ffmpeg.rb!
Error: No available formula with the name "homebrew/core/rav1e".
Please tap it and then try again: brew tap homebrew/core`

My brew config.
`HOMEBREW_VERSION: 3.3.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 96137bc19e68398ebbb7033379df288cd8b9a3f9
Last commit: 2 weeks ago
Core tap: N/A
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_FORCE_BREWED_CURL: set
HOMEBREW_INSTALL_FROM_API: set
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 13.0.0 build 1300
Git: 2.34.1 => /opt/homebrew/bin/git
Curl: 7.80.0 => /opt/homebrew/opt/curl/bin/curl
macOS: 11.6.1-arm64
CLT: 13.0.0.0.1.1630607135
Xcode: N/A
Rosetta 2: false`

**Comment by carlocab** [comment, marked as answer]
Can you try doing `brew uninstall ffmpeg` before doing `brew reinstall rav1e`?

CC @Rylan12

**Reply by dbz2k** [reply]
> Can you try doing `brew uninstall ffmpeg` before doing `brew reinstall rav1e`?
> 
> CC @Rylan12

Tried reinstalling, and actually retapping homebrew/core to see if the new install from api was the issues, but still got the same error. I do get this error on every brew upgrades now though.

`Error: Could not get runtime dependencies from /opt/homebrew/opt/ffmpeg/.brew/ffmpeg.rb!`

EDIT FIXED IT: For some reason this time when I reinstalled again rav1e it told me to do "brew link rav1e" and fixed it.

**Comment by RamblinWreck77** [comment]
Google sent me here and you are literally the only result for 

```
Library not loaded: /opt/homebrew/opt/rav1e/lib/librav1e.0.5.dylib
```

FIXED!

For future people who find this crazy obscure 1-result-on-google error:

 brew uninstall ffmpeg
 
 ... wait
 
 brew reinstall rav1e
 
 ... wait
 
 brew install ffmpeg
 
 solved the problem for me!
 
 edit: sad that brew/(?) is this broken.

**Reply by adrianoruiz** [reply]
top de mais funcionou

**Reply by MiaShopgal** [reply]
> Google sent me here and you are literally the only result for
> 
> ```
> Library not loaded: /opt/homebrew/opt/rav1e/lib/librav1e.0.5.dylib
> ```
> 
> FIXED!
> 
> For future people who find this crazy obscure 1-result-on-google error:
> 
> brew uninstall ffmpeg
> 
> ... wait
> 
> brew reinstall rav1e
> 
> ... wait
> 
> brew install ffmpeg
> 
> solved the problem for me!
> 
> edit: sad that brew/(?) is this broken.

Google cache brought me here.. XD
Thank you for replying! 
It works!!
